src/guidescanpy/flask/templates/job_query.html

```diff
@@ -193,6 +193,13 @@ <h5 class="card-header">{{ region }}</h5>
                         $('#modal-off-targets').modal('show');
                     });
 
+                    $('.datatable tbody').on('click', 'a[id="coordinate"]', function () {
```

Comment:
Do we need our own function handler here? My understanding is that by passing the appropriate flags to the `DataTable` constructor, we can do this entirely at table creation time, i.e. tweaking code around the line below:
```
                        columnDefs: [
                            {
                                target: 7,
                                visible: false
                            }
                        ],
```

Note that we needed the action handler in the other place because the `DataTable` inside the popup window was created dynamically, but that's not the case here.

I might be wrong, but this is worth a look.